sdks/java/io/xml/pom.xml

```diff
@@ -101,7 +101,7 @@
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest-core</artifactId>
-      <version>1.3</version>
+      <scope>test</scope>
```

Comment:
I think you don't have to specify test scope here, it should inherit from the dependency management in the parent pom.